docs/reference/cluster/prevalidate-node-removal.asciidoc

```diff
@@ -0,0 +1,78 @@
+[[prevalidate-node-removal-api]]
+=== Prevalidate node removal API
+++++
+<titleabbrev>Prevalidate node removal</titleabbrev>
+++++
+
+This API is currently experimental for internal use by Elastic software only.
```

Comment:
Not sure if this is meant to be removed before merge. We normally do the `{cloud-only}` warning only.